plugins/taper/run.go

```diff
@@ -79,6 +81,15 @@ func run(c *cli.Context) {
 		apiClient: apiClient,
 	}
 
+	if runConfig.EnableTraceSampling {
+		TSM, err := tsm.Create(false, runConfig.TraceSamplingManagerAddressEnv)
```

Comment:
`false` ?

Reply:
dont enable tls

Reply:
shouldn't we get it from config or it is not supported for tap?